homeassistant/components/uvc/__init__.py

```diff
@@ -1 +1,73 @@
-"""The uvc component."""
+"""The Unifi Video integration."""
+import asyncio
+import logging
+
+from uvcclient import nvr
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONF_API_KEY,
+    CONF_HOST,
+    CONF_PASSWORD,
+    CONF_PORT,
+    CONF_SSL,
+)
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS = ["camera"]
+
+
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Set up the Unifi Video component."""
+    hass.data.setdefault(DOMAIN, {})
+
+    return True
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Unifi Video from a config entry."""
+    nvrconn = await hass.async_add_executor_job(_get_nvrconn, entry)
+    hass.data[DOMAIN]["nvrconn"] = nvrconn
+    hass.data[DOMAIN]["camera_password"] = entry.data[CONF_PASSWORD]
```

Comment:
There's no need to save the password in hass.data. It's available in the config entry data.